src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -24,6 +26,9 @@ class Lambda extends RequestStreamHandler {
   val dynamoClient: DADynamoDBClient[IO] = DADynamoDBClient[IO]()
   val s3Client: DAS3Client[IO] = DAS3Client[IO]()
 
+  implicit val loggerName: LoggerName = LoggerName("Ingest Asset Opex Creator")
```

Comment:
Would it make sense to pull the logger name from the lambda function's name? I think we can get this from the `context` object.

Reply:
It makes sense. I'm going to get it from the AWS_LAMBDA_FUNCTION_NAME environment variable instead though because testing that context object is a pain.